themes/bootstrap5/templates/RecordDriver/DefaultRecord/data-summary.phtml

```diff
@@ -1,10 +1,17 @@
 <?php foreach ($this->driver->getSummary() as $summary): ?>
-  <?=$this->escapeHtml($summary) ?><br>
+  <?=$this->escapeOrCleanHtml($summary, allowHtml: true) ?><br>
 <?php endforeach; ?>
 <?php $isbn = $this->driver->getCleanISBN(); ?>
 <?php foreach ($this->summaries($isbn) as $provider => $content): ?>
-  <?php foreach ($content as $summary): ?>
-    <?php $htmlContent = str_starts_with($summary, '<') && str_ends_with($summary, '>'); ?>
-    <?=$htmlContent ? $summary : ($this->escapeHtml($summary) . '<br>')?>
-  <?php endforeach; ?>
+  <?=
+    implode(
+        '<br>',
+        array_map(
+            function ($summary) {
+                return $this->escapeOrCleanHtml($summary, allowHtml: true);
```

Comment:
Does this need to have the summary context added to match core.phtml?

Reply:
Yes, now added.